frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/uploadListItem.component.tsx

```diff
@@ -72,9 +72,17 @@ export const UploadListItem = ({
 					field: { ref, ...extras },
 				}) => (
 					<UploadListItemDestination
-						isSelected={isSelected}
+						// isSelected={isSelected}
```

Comment:
Delete this one if its not longer being used

Reply:
oops haha